### ISSUE TITLE
PortProber.pollPort makes a unexpected exception

### ISSUE THREAD

**Issue by marcos-fedato** [SeleniumHQ/selenium#2978]
## Meta -

OS:  Windows 8.1
Selenium Version:  
3.0.1 and 3.0.0
Browser:  
Tested on Firefox 48.0.2 and Chrome 53.0.2785.143 m

Browser Version:  
Firefox 48.0.2 and Chrome 53.0.2785.143 m
## Expected Behavior -

I'm new to the seleniun code but i think that the pollPort function is there to delay the process until the port is free.

maybe thats the reason why ConnectException has a catch just to ignore this error.

I'm getting the java.net.BindException inside the pollPort, and this error is not threated making the HTML Runner to shut down.

I downloaded the code and created a catch below the ConnectException catch to catch java.net.BindException and ignore it too.

Everything worked well after this, my suite runned and now i'm happy.

Maybe this is a problem for someone else

My java version:

java version "1.8.0_66"
Java(TM) SE Runtime Environment (build 1.8.0_66-b18)
Java HotSpot(TM) 64-Bit Server VM (build 25.66-b18, mixed mode)
## Steps to reproduce -

Just runned a suite inside a html file using the HTML Runner

java -jar selenium-html-runner-3.0.1.jar -htmlSuite "*chrome" "http://xxxx/xxxx/" "D:\xxxxx\xxxx.html" "D:\xxxxx\Results.html"

**Comment by lukeis** [comment]
i'm unable to reproduce the issue on OSX... going to assume it may be an issue on windows.

**Comment by barancev** [comment]
Fixed by commit ea09fb26ffb0878417149e06e33b2f5f4d8b5fe7

**Comment by snezhana-krass** [comment]
@barancev it's still reproduced on 
OS: Windows 10
Selenium Version:
3.0.1
Browser:
Firefox 48.0.2, 49.0.2

**Comment by barancev** [comment]
Yes, you have to wait for the next release.

**Comment by sisso19** [comment]
I am also experiencing this issue. Has there been set a release date for the next release?

**Comment by Littlejd1997** [comment]
Here's another one bumping, I have been trying to get some UI Automation test working, but this bug seems like the biggest delay

**Comment by cgoldberg** [comment]
If the release schedule is causing you delay, you can always build it yourself until the release is out.  The patch for this was merged into master branch almost 3 months ago.

**Comment by GrayedFox** [comment]
This bug is failing the majority of our CI builds (and it never manifests locally) - and building from source every time we build and deploy something in our CI development environment (can happen a 20-50 times in a day) is a really painful workaround.

We could build and then export the binary to a new agent, but I must say, if the nature of a bug is that it literally prevents the service from doing the very thing it is designed to do (allow people to build and run tests against it) - doesn't that sort of fall into the category of "hotfix"?

Don't mean to complain or anything, just wondering how bad a bug needs to be to break the release cycle. Thanks for the fix

**Comment by Littlejd1997** [comment]
I agree. I understand you guys have your own schedule, but so do I. I had already spent way to much time on Selenium so I ended up running Firefox with the Selenium IDE in a VM, and I added a manual approval in our CodePipeline for us to RDP into the machine and run the tests. Automating would be nice, but I can't spend anymore time on it because of course I'll run into issues building it and stuff. It always happens
